### ISSUE TITLE
Parameter parsing with JMH is broken in version 2.7

### ISSUE THREAD

**Issue by eikemeier** [jvm-profiling-tools/async-profiler#557]
Demonstration in https://github.com/eikemeier/async-profiler-example

Using JMH with `-prof "async:output=flamegraph"` and async-profiler v2.7 results in `java.lang.IllegalArgumentException: Unknown argument`. With v2.6 this still worked.

**Comment by apangin** [comment]
This is actually a problem in JMH plugin - it issues a wrong profiler command.
Before version 2.7, the unknown arguments were silently ignored, however, this was changed in #503.

I reverted the above change until JMH is fixed. Now async-profiler just prints a warning instead of failing on unknown argument.

The build with the fix is available here: https://profiler.tools/builds/async-profiler-2.7-arg-linux-x64.tar.gz
